MISampleApp/MISampleApp/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ApplicationManifest xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ApplicationTypeName="MISampleAppType" ApplicationTypeVersion="1.0.0" xmlns="http://schemas.microsoft.com/2011/01/fabric">
+  <Parameters>
+    <Parameter Name="MISampleConsole_InstanceCount" DefaultValue="-1" />
+    <Parameter Name="MISampleWeb_ASPNETCORE_ENVIRONMENT" DefaultValue="" />
+    <Parameter Name="MISampleWeb_InstanceCount" DefaultValue="-1" />
+  </Parameters>
+  <!-- Import the ServiceManifest from the ServicePackage. The ServiceManifestName and ServiceManifestVersion 
+       should match the Name and Version attributes of the ServiceManifest element defined in the 
+       ServiceManifest.xml file. -->
+  <ServiceManifestImport>
+    <ServiceManifestRef ServiceManifestName="MISampleConsolePkg" ServiceManifestVersion="1.0.0" />
+    <ConfigOverrides />
+    <Policies>
+      <ContainerHostPolicies CodePackageRef="Code" Isolation="hyperv" RunInteractive="true">
+        <!-- See https://aka.ms/I7z0p9 for how to encrypt your repository password -->
+        <RepositoryCredentials AccountName="my cr username" Password="my cr password" PasswordEncrypted="false" />
+        <!-- < PortBinding ContainerPort="81" EndpointRef="MISampleConsoleTypeEndpoint" /> -->
```

Comment:
You can remove commented entries (meaning those which aren't comments/explanatory.)